templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -5,21 +5,29 @@ const initNavigationSliding = () => {
   const toggles = document.querySelectorAll('.p-navigation__nav .p-navigation__link[aria-controls]:not(.js-back-button)');
   const searchButtons = document.querySelectorAll('.js-search-button');
   const menuButton = document.querySelector('.js-menu-button');
-  const dropdownNavLists = document.querySelectorAll('.js-dropdown-nav-list');
-  const topNavList = [...dropdownNavLists].filter((list) => !list.parentNode.closest('.js-dropdown-nav-list'))[0];
+  const dropdownNavLists = document.querySelectorAll('.p-navigation__dropdown');
```

Comment:
We are not very consistent with this, but we were trying to avoid using styling CSS class names in JS.

So in this case having `js-dropdown-nav-list` was meant to allow having a specific pointers for JS, that are not connected to CSS styles. 
Ideally we want to avoid using `p-` class names in JS, and either use custom `js-` class names, or other ways to select items (via DOM, parents, data attributes, aria attributes, etc).